spec/index.bs

```diff
@@ -4305,9 +4303,8 @@ enum GPUInputStepMode {
 </script>
 
 <script type=idl>
-dictionary GPUVertexStateDescriptor {
-    GPUIndexFormat indexFormat;
-    sequence<GPUVertexBufferLayoutDescriptor?> vertexBuffers = [];
+dictionary GPUVertexState: GPUProgrammableStage {
+    sequence<GPUVertexBufferLayoutDescriptor?> inputs = [];
```

Comment:
I understand the concern (input may seem like it refers to bind groups). And I think `inputBuffers`/`bufferInputs` may be equally confusing (buffers can also be in bind group). Not sure what a good name for this would be... we need to keep in mind that this is a "hot path", i.e. something that every single descriptor writer would need to type.

Reply:
I independently had a similar concern when reading the full descriptor you posted above: `inputs` make it sound like it is a list of things that correspond to the `var<in>` in the shader, while it is a list of list of these things. That's why I have a slight preference for `buffers` because everyone understands what `vertex.buffers` means and its close relationship with `setVertexBuffers`.

Reply:
I think it's okay to be slightly redundant in naming here for clarity. I don't think there will be SO many descriptors in a program that it would be arduous to type a few letters here. vertex.vertexBuffers, vertex.vertexData? I'm also fine with vertex.inputs as well, I don't have the same concern as Corentin though I have a tiny concern that it sounds like it would include uniform buffers.

Reply:
ok, let's chat about the naming of these fields somewhere, and get it landed